pom.xml

```diff
@@ -6,11 +6,11 @@
 
     <packaging>pom</packaging>
 
-    <groupId>com.exadel.aem</groupId>
-    <artifactId>aem-backpack</artifactId>
-    <version>1.1.0</version>
+    <groupId>com.exadel.etoolbox</groupId>
+    <artifactId>etoolbox-backpack</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
 
-    <name>AEM BackPack Tool</name>
+    <name>EToolbox BackPack</name>
     <description>Automates generating rich and responsive authoring interfaces for Adobe Experience Manager (TM) via Maven-powered build and deployment workflow</description>
```

Comment:
Please change this description as it comes from the Toolkit ;)

Reply:
You also need to make changes to the `<developer>` section to reflect MarTech and not EDMTP